src/foam/nanos/fs/File.js

```diff
@@ -202,9 +206,19 @@ foam.CLASS({
       name: 'labels',
       class: 'StringArray',
       documentation: 'List of labels applied to this file',
+      validationPredicates: [
+        {
+          args: ['labels'],
+          predicateFactory: function(e) {
+            return e.NOT(e.CONTAINS(foam.nanos.fs.File.LABELS, ""));
+          },
+          errorString: 'INVALID_FILE_LABEL'
+        }
+      ],
```

Comment:
This validation predicate seems to fail every time 
I am trying to block "" being assigned as a label (if you add a choiceview but do not select a value)

Reply:
Yeah appears correct. So failing == the errorString fails to popup if file label is an empty string? Also what would the default be then if you add a choiceview but do not select a value?

Reply:
Resolved in slack and zoom.